etherscanapi/src/main/java/jfyg/queries/QueryMediator.kt

```diff
@@ -0,0 +1,63 @@
+package jfyg.queries
+
+import android.util.Log
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.Disposable
+import io.reactivex.schedulers.Schedulers
+import jfyg.ApiKey
+import jfyg.network.RestClient
+import jfyg.response.account.AccountResponse
+import jfyg.response.stat.StatResponse
+
+/**
+ * A mediator between the responses and errors that come from every query
+ */
+class QueryMediator : AccountQueries, StatQueries {
+    private val TAG = javaClass.name
+
+    private var statInfo = StatResponse()
+    private var accountInfo = AccountResponse()
+
+    override fun accounts(module: String?,
+                          action: String?,
+                          txhash: String?,
+                          address: String?,
+                          statblock: String?,
+                          endblock: String?,
+                          offset: String?,
+                          sort: String?,
+                          blocktype: String?,
+                          tag: String?): Disposable? =
+            RestClient().getQuery()
+                    .getAccounts(module, action, txhash, address, statblock, endblock, offset, sort, blocktype, tag, ApiKey
+                            .takeOff
+                            .callApiKey())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribeOn(Schedulers.io())
+                    .subscribe(this::handleResponse, this::handleError)
+
+    override fun stats(module: String, action: String): Disposable? =
+            RestClient().getQuery()
+                    .getStats(module, action, ApiKey.takeOff.callApiKey())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribeOn(Schedulers.io())
+                    .subscribe(this::handleResponse, this::handleError)
+
+
+    override fun handleResponse(response: AccountResponse) {
```

Comment:
This handleresponse() isn't being triggered.  The network call is executing correctly.. i've monitored the requests and the responses and they are going through fine according to charlesProxy..